frontend/public/components/events.jsx

```diff
@@ -64,14 +64,23 @@ const kindFilter = (reference, { involvedObject }) => {
   if (reference === 'all') {
     return true;
   }
-
-  if (isGroupVersionKind(reference)) {
-    const group = apiGroupForReference(reference);
-    const kind = kindForReference(reference);
-    const { group: eventGroup } = groupVersionFor(involvedObject.apiVersion);
-    return involvedObject.kind === kind && eventGroup === group;
-  }
-  return involvedObject.kind === reference;
+  let filterKind = false;
+  const kinds = reference.indexOf(',') > 0 ? reference.split(',') : [reference];
+  kinds.map((ref) => {
```

Comment:
`Array.some()` is better here

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

```ts
return kinds.some((ref: string) => {
  if (isGroupVersionKind(ref)) {
    const group = apiGroupForReference(ref);
    const kind = kindForReference(ref);
    const { group: eventGroup } = groupVersionFor(involvedObject.apiVersion);
    return involvedObject.kind === kind && eventGroup === group);
  }

  return involvedObject.kind === ref;
};
```